frontend/src/components/ViewActivities/activityfns.js

```diff
@@ -54,4 +57,30 @@ export function getField(activity, fieldName, defaultValue){
     return defaultValue;
   }
   return activity[fieldName];
+}
+
+/**
+ * Write contents into an activity in the database.
+ * 
+ * @param {string} tripId Database ID of the trip whose actiivty should be modified.
+ * @param {string} activityId Database ID of the activity to be modified.
+ * @param {Object} newValues Dictionary of the new values in {fieldName: newValue} form
+ */
```

Comment:
include the @return {boolean} ...

Reply:
+1, this should include the return type "{boolean}" rather than just "true."